### ISSUE TITLE
Feature request: deleting file is not deleting buffer 

### ISSUE THREAD

**Issue by Migacz85** [francoiscabrol/ranger.vim#77]
I found out that when I had opened buffers, I moved files from one directory to another. 
Vim was unaware of this and buffers of deleted files were still there. And this produced confusion. 
Is there a way that after deleting or moving files buffers will be deleted aswell ? 
It would be perfec also if fugitive will also know that files were deleted or moved. 

Is that possible to implement this ? Ranger is just brilliant to manipulating files. It would be perfect to implement this well with vim

**Comment by francoiscabrol** [comment]
Yes I understand the problem, it's a duplicate of #72. However I don't see how to make vim aware of outside changes...

**Comment by Migacz85** [comment]
Ok, Is this even possible to run some vim function when the ranger is turned on ?

**Comment by francoiscabrol** [comment]
In neovim yes but it's not user friendly. You can still switch to the vim's command mode with `esc` and then run any vim command while ranger stays opened in a terminal buffer. Then you can come back to ranger by re-switching to the insert mode by pressing `i`. But switching between neovim's modes while ranger is opened is not a nice flow.
With the regular vim, you can't go back to vim without closing ranger.
